contents/blog/best-statsig-alternatives.md

```diff
@@ -0,0 +1,454 @@
+
+It's a popular tool – according to BuiltWith, as of April 2024 it's used by 8,981 of the top 1 million websites. It's grown noticeably in the last 12 months or so, likely due to customers migrating from Google Optimize, which was shut down in September 2023.
+
+#### Key features
+
+- 🧪 **A/B testing:** Improve conversion with web, mobile, and server-side A/B testing.
+
+- 💽 **Data platform:** Collect and analyze custom data across your stack.
+
+- 🤔 **Insights:** Understand your users with session recordings, heatmaps, analytics, and surveys.
+
+- 👔 **Personalization:** Create and tailor user journeys and campaigns to the audience, location, and time.
+
+- 📋 **Planning:** Ideate and plan optimization campaigns in one location.
+
+### How does VWO compare to Statsig?
+
+VWO is closer to Optimizely than Statsig, though it offers most of the same features. It doesn't offer a complete product analytics tool, but it does offer basic funnel analysis, heatmaps, session replays, and user surveys as part of its wider platform.
+
+<ComparisonTable column1="VWO" column2="Statsig">
+  <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Deploy features safely with targeting and percentage rollouts" />
+  <ComparisonRow column1={true} column2={true} feature="Local evaluation" description="Use local, cached flag values to increase speed" />
+  <ComparisonRow column1={true} column2={true} feature="Bootstrapping" description="Flags available on frontend application load" />
+  <ComparisonRow column1={false} column2={false} feature="Payloads" description="Flags with string, number, or JSON payloads" />
+  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run tests and see the impact of changes with custom goals and reports" />
+  <ComparisonRow column1={true} column2={true} feature="Multivariate (A/B/n) testing" description="Test multiple variants of a change" />
+  <ComparisonRow column1={true} column2={true} feature="Secondary metrics" description="Monitor impact on unrelated metrics" />
+  <ComparisonRow column1={true} column2={true} feature="Multi-armed bandit" description="Optimize tests automatically by allocating traffic to the best performing variant." /> 
+  <ComparisonRow column1={true} column2={true} feature="Mutually exclusive experiments" description="Isolate user groups for simultaneous, independent experiments." />
+  <ComparisonRow column1={true} column2={false} feature="No code experiments " description="Create A/B tests using a visual editor" />
+  <ComparisonRow column1={false} column2={true} feature="Product analytics" description="Track trends, funnels, user paths, and retention" />
+  <ComparisonRow column1={false} column2={false} feature="Autocapture" description="Capture events without manual logging" />
```

Comment:
I counted VWO as having autocapture in https://posthog.com/blog/best-vwo-alternatives but it is a bit [ambiguous](https://vwo.com/insights/#features)

Reply:
What actually implies they do have it? I couldn't see anything. Tried their docs as well, but they're basically impossible to navigate!

Reply:
I think this implies no, because if they did have it then you would already by collecting events, rather than "ready to collect events".

![Screenshot 2024-04-30 at 16 50 03](https://github.com/PostHog/posthog.com/assets/92976667/cb77f5d0-c415-4b76-a3e4-84f209d339c6)